.idea/jsLinters/eslint.xml

```diff
@@ -2,5 +2,6 @@
 <project version="4">
   <component name="EslintConfiguration">
     <option name="fix-on-save" value="true" />
+    <files-pattern value="{**/*,*}.{js,ts,jsx,tsx,html,vue,gohtml}" />
```

Comment:
Vue?

Reply:
This was auto-generated after I changed some settings, not sure which tbh, but I think it's harmless